### ISSUE TITLE
List, Export and Import followed hashtags

### ISSUE THREAD

**Issue by djhenderson** [mastodon/mastodon#22921]
### Pitch

We can now follow hashtags from the browser client, however there is no capability to list, export or import followed hashtags.

### Motivation

This request completes the feature to follow hashtags.

We can link to tags with a URL of the form `HOST/tag/HASHTAG` where `HOST` is the Mastodon server's FQDN, `HASHTAG` is the part of the hashtag following the octothorp character, and `tag` is a literal.

URLs using this pattern could be listed with followed accounts, exported with followed accounts, and imported to followed accounts or wherever followed hashtags are stored.

For completeness, these URLs should be accepted on the explore page by making them equivalent to the corresponding hashtag.

**Comment by estelle4565** [comment]
Yes please. Personally do not care much about the mechanics of  it. I just want to be able to import my followed hashtags when I move to a new account.

**Comment by BDJMasto** [comment]
Being able to export/import hashtags as their own CSV will also allow for the sharing of hashtag "collections" with others.

**Comment by expertmanofficial** [comment]
I'm supporting that issue. Right now, I'm moving an account from mstdn.social to defcon.social, and now I currently re-following hashtags manually, which is uncomfortable :-(

**Comment by brendanjones** [comment]
This is duplicate of https://github.com/mastodon/mastodon/issues/21091.

**Comment by novoid** [comment]
With the help of Claude Code, I just implemented "kamm": https://codefloe.com/publicvoit/kamm

It is able to retrieve a list of hashtags you follow and stores in a local CSV file.

Furthermore, it is able to follow new hashtags from a CSV file you provide. This list may be even large.
I've tested it with over 800 tags as I'm using hashtags for comments on my blog articles. I've described the idea on https://karl-voit.at/2026/08/22/Blog-Comments-via-Hashtags/

HTH
